tests/lib/thrift/ThriftTest/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Young Mei
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(thrift_test)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/../../../../cmake/thrift.cmake)
+
+FILE(GLOB app_sources
+    src/*.cpp
+    src/thrift/server/*.cpp
```

Comment:
Rather than copying `ThriftTest.thrift` and `server.cpp` into the project, let's just reuse the existing sources, similar to what was done here:

https://github.com/zephyrproject-rtos/gsoc-2022-thrift/blob/main/lib/thrift/CMakeLists.txt#L7

Reply:
I don't think it's possible to reuse `server.cpp` from the upstream - it includes quite some boost headers, and headers for protocols/transports in thrift that not yet ported (which further includes more external headers that not in zephyr yet).